samples/bluetooth/iso_broadcast_benchmark/prj.conf

```diff
@@ -9,6 +9,7 @@ CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_ISO_SYNC_RECEIVER=y
 CONFIG_BT_ISO_MAX_CHAN=1
 CONFIG_BT_ISO_TX_BUF_COUNT=2
+CONFIG_BT_ISO_TX_MTU=230
```

Comment:
230 is odd value to use, add a comment on why/how to arrive at this value.

Reply:
This was actually caused by an issue found and fixed in https://github.com/zephyrproject-rtos/zephyr/pull/43019